RecoLocalCalo/HGCalRecAlgos/interface/HGCal3DClustering.h

```diff
@@ -23,17 +23,30 @@ class HGCal3DClustering
   {
   }
 
- HGCal3DClustering(const edm::ParameterSet& conf, edm::ConsumesCollector& sumes, const std::vector<double>& radii_in, uint32_t min_clusters) :
-  radii(radii_in),
-  minClusters(min_clusters),
-  points(2*(maxlayer+1)),
-  minpos(2*(maxlayer+1),{ {0.0f,0.0f} }),
-  maxpos(2*(maxlayer+1),{ {0.0f,0.0f} }),
-  es(0),
-  zees(2*(maxlayer+1),0.),
-  clusterTools(std::make_unique<hgcal::ClusterTools>(conf,sumes))
-  {
-  }
+  HGCal3DClustering(const edm::ParameterSet& conf, edm::ConsumesCollector& sumes, const std::vector<double>& radii_in, uint32_t min_clusters) :
+   radii(radii_in),
+   minClusters(min_clusters),
+   points(2*(maxlayer+1)),
+   minpos(2*(maxlayer+1),{ {0.0f,0.0f} }),
+   maxpos(2*(maxlayer+1),{ {0.0f,0.0f} }),
+   es(0),
+   zees(2*(maxlayer+1),0.),
+   clusterTools(std::make_unique<hgcal::ClusterTools>(conf,sumes))
+   {
+   }
+
+
+   HGCal3DClustering(const edm::ParameterSet& conf, edm::ConsumesCollector& sumes) :
+    radii(conf.getParameter<std::vector<double>>("multiclusterRadii")),
+    minClusters(conf.getParameter<unsigned>("minClusters")),
```

Comment:
this can be replaced by 
```c++
: HGCal3DClustering(conf, sumes, conf.getParameter<std::vector<double>>("multiclusterRadii"),
            conf.getParameter<unsigned>("minClusters") {}
```

Reply:
> this can be replaced by
> 
> ```c++
> : HGCal3DClustering(conf, sumes, conf.getParameter<std::vector<double>>("multiclusterRadii"),
>             conf.getParameter<unsigned>("minClusters") {}
> ```

please use this to reduce code replication